zerver/openapi/zulip.yaml

```diff
@@ -7064,6 +7189,48 @@ paths:
                         description: |
                           Unique ID of the scheduled message reminder.
                     example: {"msg": "", "reminder_id": 42, "result": "success"}
+
+  /reminders/{reminder_id}:
+    delete:
+      operationId: delete-reminder
+      tags: ["reminders"]
+      summary: Delete a reminder
+      description: |
+        Delete, and therefore cancel sending, a previously [scheduled
+        reminder](/help/schedule-a-reminder).
+
+        **Changes**: New in Zulip 11.0 (feature level ZF-f1fd6c).
+      parameters:
+        - name: reminder_id
+          in: path
+          schema:
+            type: integer
+          description: |
+            The ID of the reminder to delete.
+
+            This is different from the unique ID that the reminder would have
+            after being sent.
```

Comment:
This is confusing -- I think some unfortunate find/replace on the "ID that the message would have ..."

Reply:
Done